rust/hyperlane-core/src/types/chain_data.rs

```diff
@@ -11,6 +13,15 @@ pub struct BlockInfo {
     pub number: u64,
 }
 
+/// Metrics about the chain.
+#[derive(Debug, Clone, Default, new)]
+pub struct ChainInfo {
+    /// Information about the latest block
+    pub latest_block: BlockInfo,
+    /// The current gas price, in the lowest denomination (e.g. gwei)
```

Comment:
nit: gwei -> wei if we're doing lowest denomination now

Reply:
my assumption is that we are using low denomination, since I'm now scaling with `decimals_by_protocol(...)`, let me know if not